R/plot_grid.R

```diff
@@ -135,6 +135,10 @@ plot_grid <- function(..., plotlist = NULL, align = c("none", "h", "v", "hv"),
 
   # Make a list from the ... arguments and plotlist
   plots <- c(list(...), plotlist)
+  # For pipe-friendlyness, also handle passing a list of plots as only positional argument
+  if (is.null(plotlist) && length(plots) == 1L && identical(class(plots[[1L]]), "list")) {
```

Comment:
Currently, this is maximally strict. If you want, I can relax it, e.g. by removing `is.null(plotlist)` and/or replacing `identical(class(plots[[1L]]), "list")` with `!("gg" %in% class(plots[[1L]]))`. I’m not sure if the second one is correct however: does `plot_grid` accept things that don’t have the class `gg`?